README.md

```diff
@@ -92,6 +92,7 @@ If you do not have a GPU, add the define "USE_CPU_ONLY", for example
 by adding -DUSE_CPU_ONLY=1 to the cmake command line.
 * Optional: BLAS Library: OpenBLAS (libopenblas-dev) or Intel MKL
 * The program has been tested on Windows, Linux and macOS.
+* To work successfully, autogtp binary needs to also have leelaz binary to be in the same directory. The below instructions can compile leelaz, and optionally autogtp at the same time if you want. However if for some reason you want to only compile autogtp binary (without leelaz), see [the autogtp readme](https://github.com/gcp/leela-zero/tree/next/autogtp).
```

Comment:
Why would someone need to compile AutoGTP separately if he isn't a developer? If they are, they can surely find the README for AutoGTP?

Reply:
Also, this doesn't actually link to the README, and it preferably should use a relative link so it works when merged to master?

Reply:
ok, i'll use a direct link to master

if you want, i can also remove this part

Reply:
please have a look at the new autogtp readme and review again : https://github.com/gcp/leela-zero/blob/next/autogtp/README.md